src/sass/shared/variables.scss

```diff
@@ -58,13 +61,15 @@ $font-size-footer-titles: 21px;
 $font-size-footer-text: 16px;
 $font-size-pill-button: 21px;
 $font-size-terminal-source-code: 14px;
+$font-size-signature: 48px;
 
 // Font families
 $font-roboto: 'Roboto', sans-serif;
 $font-source-sans-pro: 'Source Sans Pro', sans-serif;
 $font-source-code: 'Hack', 'monaco', monospace;
 $font-default: $font-source-sans-pro;
 $font-titles: $font-roboto;
+$font-handwritten: 'Caveat', cursive;
```

Comment:
Following your initial organization for font declaration, the font family declaration should be done first, and then used for `$font-handwritten`
something like:
```
$font-roboto: 'Roboto', sans-serif;
$font-source-sans-pro: 'Source Sans Pro', sans-serif;
$font-source-code: 'Hack', 'monaco', monospace;
$font-caveat: 'Caveat', cursive; // <- family definition

$font-default: $font-source-sans-pro;
$font-titles: $font-roboto;
$font-handwritten: $font-caveat; // <- lets use the font
```